README.md

```diff
@@ -270,30 +270,65 @@ Need help or want to contribute ideas or code? Head over to our [CONTRIBUTING](C
-✔ tagging release v0.18.2
-ℹ Run `git push --follow-tags origin master && npm publish` to publish
+### Preparation
+
+1. Checkout a branch `name-v5-0-1`. When deciding what version will be released it is important to look over the past commits since the last release and the updated dependencies release notes in step 2 to see if this will be a major, minor, or patch release. If you have any questions feel free to ask the maintainers.
+
+2. Ensure we have the latest polkadot-js dependencies
+
+  * @polkadot/api [release notes](https://github.com/polkadot-js/api/releases)
+  * @polkadot/apps-config [release notes](https://github.com/polkadot-js/apps/releases)
+  * @polkadot/util-crypto [release notes](https://github.com/polkadot-js/common/releases)
+
+  Note: Every monday the polkadot-js ecosystem will usually come out with a new release. It's important that we keep up, and read the release notes for any breaking changes, or high priority updates.
+
+3. After updating the dependencies, the next step is making sure the release will work against all noted runtimes for Polkadot, Kusama, and Westend. This can be handled via the [sidecar-runtime-test](https://github.com/TarikGul/sidecar-runtime-test) helper library. Instructions for how to run it are in the repos README.md. Before moving forward ensure all tests pass, and if it warns of any missing types feel free to make an issue [here](https://github.com/paritytech/substrate-api-sidecar/issues). 
+
+4. Update the version in the package.json (this is very important for releasing on NPM).
+
+5. Update the changelog by looking at the past commits since the last release. Follow the format of previous releases. 
+
+  * Make sure to note if it has a high upgrade priority.
+
+6. Commit with ex: `chore(release): 5.0.1`, then push your release up, make a PR, get review approval, then merge. 
+
+  * NOTE: Before pushing up as a sanity check run the 3 following commands and ensure they all run with zero errors. There is one exception with `yarn test` where you will see an `InternalServerError`, that is alright. As long as all the test suites pass. 
+  ```bash
+  $ yarn test
+  $ yarn build
+  $ yarn lint
+  ```
+
+### Release && Pubishing to NPM and Github
+
+** NPM **
+
+NOTE: You must be a member of the @substrate NPM org and must belong to the developers team within the org. (Please make sure you have 2FA enabled.) 
+
+1. Now that master has the commit for the release, pull down master. Run `yarn build` to ensure the target JS build is there for the NPM release. 
+
+2. Run the following commands. (Please ensure you have 2FA enabled)
+
+```bash
+$ yarn npm login
+...
+$ yarn npm publish
 ```
 
-To publish the new package, just follow the instructions: `git push --follow-tags origin master && npm publish.`
-You must have access to the @substrate organization on npm to publish.
+** Github **
+
+1. Make sure the tag reflects your corresponding version, and run:
+
+```bash
+git tag v5.0.1
+git push origin --tags
```

Comment:
```suggestion
git push origin v5.0.1
```

Reply:
Ahh yes, the ole single tag push. Nice reminder, thanks!